disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/storage/responses/WritableResult.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.disk.buffering.internal.storage.responses;
+
+public enum WritableResult {
+  SUCCEEDED,
+  CLOSED,
```

Comment:
this enum value is returned, but no caller reacts on it

Reply:
Thanks, it's updated now.